src/test/java/io/gravitee/policy/jws/JWSPolicyTest.java

```diff
@@ -240,13 +240,20 @@ public void shouldValidateCRL() throws Exception {
     @Test(expected = CertificateException.class)
     public void shouldValidateCRL_certificateException() throws Exception {
         CertificateFactory certificateFactory = CertificateFactory.getInstance("X.509");
-        String pemPath = getPemFilePath("wikipedia.pem");
+        String pemPath = getPemFilePath("revoked.pem");
         String fileContent = new String(Files.readAllBytes(Paths.get(pemPath)), Charset.forName(StandardCharsets.UTF_8.name()));
         X509Certificate cert = (X509Certificate) certificateFactory.generateCertificate(
             new ByteArrayInputStream(fileContent.getBytes(StandardCharsets.UTF_8))
         );
-        // WARNING: this test is based on real certificate from Wikipedia. Depending on the state of the expired certificates entries, it could be broken.
-        jwsPolicy.validateCRLSFromCertificate(cert, new BigInteger("18135611436097276018781887707242686781"));
+        /*
+         * WARNING:
+         * this test is based on a fake certificate from https://badssl.com/. (See also https://github.com/chromium/badssl.com)
+         * As this certificate will expire, it will need to be regenerated. Here is how to do it:
```

Comment:
When this fake certificate expires, the test won't fail as it throws the same Exception, so we won't be to noticed to regenerate it.
Maybe we could catch the `CertificateException` and check the error message ?

Could we also get a dedicated `expired certificate` test case ?

Reply:
- I replaced the *revoked.pem* file with the right one (it was not well generated)
 - I added an expiration test in the revocation unit test to be sure that the revoked certificate is not expired.

Reply:
great, thanks